Sage/carbon
# feat: support new `Sage UI` typeface

Addresses FE-5028

BREAKING CHANGE: @sage/design-tokens package is now a peer dependency instead of a built-in dependency.

### Proposed behaviour

- Remove `Lato` typeface from storybook assets
- Remove all references to `Lato` in source code, replace with `Sage UI`
- Add `@font-face` definitions for `Sage UI` and `CarbonIcons` in `src/style/fonts.css`
   - The three `Sage UI` typeface files for regular, medium and bold are loaded in individually via CDN
   - Storybook preloads the typeface files and loads `fonts.css` on startup
- Update to the latest version of `@sage/design-tokens`
- Change `@sage/design-tokens` from a required dependency to a development/peer dependency.
    - This is a **breaking change**.
    - Since the package is separate project to Carbon, this prevents consumers having to upgrade Carbon every time we update the package ourselves and gives them flexibility when it comes to upgrading.
- Update Getting Started documentation on font setup and theming details

_Note: The codesandbox template will be updated to reflect these changes once this PR is released._

![Screenshot 2022-03-23 at 14 12 06](https://user-images.githubusercontent.com/18368713/159719497-e4ee2f4d-4777-4f98-806f-a9db1a806784.png)

### Current behaviour

Carbon supports the `Lato` typeface:

![Screenshot 2022-03-23 at 14 12 30](https://user-images.githubusercontent.com/18368713/159719543-d1d30b25-7f15-4d62-ae56-ea49592fc026.png)

### Checklist




#### QA


### Testing instructions



Create a local copy of this branch via `git pr 4992` and start storybook. Check storybook loads the `Sage UI` typeface correctly.
